Main.java

```diff
@@ -0,0 +1,43 @@
+import java.util.Scanner;
+
+class Main {
+    public static int sum(int number) {
+        int sum = 0;
+        while (number > 0) {
+            sum = sum + number % 10;
+            number = number / 10;
+        }
+        return sum;
+    }
+
+    public static String sum(String word, String text) {
```

Comment:
It's ok but more logical to write 'int' instead of 'String' as the return type and
return len1 + len2